proto/commonfate/control/config/v1alpha1/slack_alert.proto

```diff
@@ -30,6 +31,8 @@ message SlackAlert {
   optional string integration_id = 5;
   bool use_web_console_for_approve_action = 6;
   bool send_direct_messages_to_approvers = 7;
+  bool use_web_console_for_activate_action = 8;
+  bool use_web_console_for_close_action = 9;
```

Comment:
@ckluy31 I see you followed the pattern from above that was introduced with `use_web_console_for_approve_action`, I am thinking though it might actually be better here to have a variable called `disable_interactivity_handlers`.

```suggestion
  // Disables all webhook handlers for the Slack integration.
  // All buttons in the Slack integration will link the user to Common Fate URLs
  // rather than being directly interactive.
  // This is useful when deploying Common Fate in a VPC
  // with restricted network ingress.
  //
  // If set to true, this setting takes priority over 'use_web_console_for_activate_action'.
  bool disable_interactivity_handlers = 8;
```

For the use case where Common Fate is deployed behind a private VPC, we need to disable all of the interactive buttons and also disable the `/access` command.

This will allow us to achieve the two main use cases that we're looking at:

1. If a team wants to enforce SSO for access approvals, they will set `use_web_console_for_approve_action=true`
2. If a team wants to deploy Common Fate to a VPC without any public ingress, they will set `disable_interactivity_handlers=true`.